content/en/blog/cncf-contribution-guide/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Contributing to a CNCF Project: A Comprehensive Guide for Newcomer's"
```

Comment:
Do we have to say specifically CNCF project?. can it be changed to OpenSource?

Reply:
done

Reply:
If i change CNCF word  to open source then  this blog would differ bit away from cncf. Since, we are promoting and using the cncf product., So, i think there is no harm in using naming.  
And i agree with you POV because there is much more in opensource beyound cncf. 

What do you think ?

Reply:
IMO , we could add some line like what is open source project ? Then talk about biggest home for such project and how to contribute to it ( as it is already there )

Reply:
I would like to keep the topic generic instead of tieing with any vendor or org focus.

Reply:
I would like to change this article to how to get to OpenSource?

Reply:
we could have title like "Navigating the World of Open Source: A Newcomer's Guide to CNCF"

and we can remove the description present in image and might use other cover photo